Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2144 @@
+            client_id=username,
+            client_secret=password
+        )
+
+        args = demisto.args()
+
+        demisto.debug(f'Command being called is {demisto.command()}')
+
+        commands = {
+            # Command Status
+            "symantec-edr-endpoint-status": get_endpoint_status_command,
+
+            # Domain File Associations
+            "symantec-edr-domain-file-association-list": get_domain_file_association_list_command,
+
+            # Endpoint Domain Associations
+            "symantec-edr-endpoint-domain-association-list": get_endpoint_domain_association_list_command,
+
+            # Endpoint File Associations
+            "symantec-edr-endpoint-file-association-list": get_endpoint_file_association_list_command,
+
+            # Get Incidents
+            "symantec-edr-incident-list": get_incident_list_command,
+
+            # Events For Incidents
+            "symantec-edr-incident-event-list": get_event_for_incident_list_command,
+
+            # Get Incident Comments
+            "symantec-edr-incident-comment-get": get_incident_comments_command,
+
+            # Patch Incidents Command to (Close Incidents, Update Resolution or Add Comments)
+            "symantec-edr-incident-update": patch_incident_update_command,
+
+            # System Activities
+            "symantec-edr-system-activity-list": get_system_activity_command,
+
+            # Audit Events
+            "symantec-edr-audit-event-list": get_audit_event_command,
+
+            # Allow List Policies
+            "symantec-edr-allow-list-policy-get": get_allow_list_command,
+
+            # Deny List Policies
+            "symantec-edr-deny-list-policy-get": get_deny_list_command,
+
+            # Domain Instances
+            "symantec-edr-domain-instance-list": get_domain_instance_command,
+
+            # Endpoint Instances
+            "symantec-edr-endpoint-instance-list": get_endpoint_instance_command,
+
+            # File Instances
+            "symantec-edr-file-instance-list": get_file_instance_command,
+
+            # Events
+            "symantec-edr-event-list": get_event_list_command,
+
+        }
+        if command == "test-module":
+            return_results(test_module(client))
+        elif command == 'fetch-incidents':
+            return_results('ok')
+        elif command == "symantec-edr-endpoint-isolate":
+            return_results(get_endpoint_command(client, args, 'isolate_endpoint'))
+        elif command == "symantec-edr-endpoint-rejoin":
+            return_results(get_endpoint_command(client, args, 'rejoin_endpoint'))
+        elif command == "symantec-edr-endpoint-delete-file":
+            return_results(get_endpoint_command(client, args, 'delete_endpoint_file'))
+        elif command == "symantec-edr-endpoint-cancel-command":
+            return_results(get_endpoint_command(client, args, 'cancel_command'))
+        elif command in commands:
+            return_results(commands[command](client, args))
```

Comment:
please only use `return_results` once in a file.

Reply:
Fixed